config/ember-try.js

```diff
@@ -50,11 +50,6 @@ module.exports = async function () {
       },
       {
         name: 'ember-default-with-jquery',
-        env: {
-          EMBER_OPTIONAL_FEATURES: JSON.stringify({
-            'jquery-integration': true,
-          }),
-        },
         npm: {
           devDependencies: {
             '@ember/jquery': '^1.1.0',
```

Comment:
After dropping ember-fetch, when `'jquery-integration': true`, tests regarding FormData started failing - specifcally, this assert on this test for each adapter
https://github.com/adopted-ember-addons/ember-data-factory-guy/blob/f1f2a37056f182b367f58247636f2e0c15b3be06/tests/unit/shared-adapter-behaviour.js#L1430-L1436

The FormData just wouldn't come through in any form. I am suspecting that this issue from pretender is kicking in https://github.com/pretenderjs/pretender/issues/159

Either way, `'jquery-integration': true` hasn't been supported in ember for a while now, so I think we are okay to drop support for it. It'll still "work" for people too if they want, as long as they don't use FormData.

I've left the ember-try scenario since it still works fine with `@ember/jquery`